gameplan/hooks.py

```diff
@@ -229,3 +229,24 @@
 # Recommended only for DocTypes which have limited documents with untranslated names
 # For example: Role, Gender, etc.
 # translated_search_doctypes = []
+
+frappe_search_doctypes = {
+    "GP Discussion": {
+        "title": "title",
+        "content": ["content"],
```

Comment:
what does the list mean here? can I put multiple content fields? how will that work? will it concat the fields and index?

Reply:
Yep, it'll concat the fields with `|||`. It's the client's job to replace that with something (I've done a dot here).